src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/FileUtils.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+import java.io.*;
+import java.util.Iterator;
+
+public final class FileUtils {
+
+    public static void writeLinesToSingleFile(final Iterator<String> linesToWrite, final String fileName) {
```

Comment:
Thanks for taking a look, @pshapiro4broad. This would also us to write to local FS, hadoop, and google buckets, i.e. more general than `Files.write()`.